codecov.yml

```diff
@@ -0,0 +1,24 @@
+# Codecov configuration for space_packet_parser
+# https://docs.codecov.com/docs/codecovyml-reference
+
+coverage:
+  status:
+    project:
+      default:
+        target: 95%  # Target coverage percentage
+        threshold: 5%  # Allow 5% drop without failing
+        if_no_uploads: error  # Fail if no coverage data
+        only_pulls: false  # Check coverage on all commits
+    patch:
+      default:
+        target: 85%  # Lower threshold for new code
+        threshold: 5%
+        only_pulls: true  # Only check patch coverage on PRs
+
+  ignore:
+    - "tests/**/*"  # Don't include test files in coverage
```

Comment:
I have heard it argued that this is actually the most important coverage of all :) If your tests aren't actually running, then you are in trouble! (If you write the same test_my_feature() function in the same file on accident one will shadow the other and not actually get run.

Reply:
Good point!